components/ui-fragments/Input.js

```diff
@@ -18,12 +18,20 @@ const SearchInputWrapper = styled.div`
 const HalfInput = styled(Input)`
   border-radius: 10px 0 0 10px;
   flex: 1;
+  :focus {
+    outline: none;
+    box-shadow: 0px 0px 10px 2px ${({ theme }) => theme.colors.border};
```

Comment:
This shadow is used a lot. It might be a good idea to place it in the theme. Like
```js
...
 shadows: { outline: '0px 0px 10px 2px #SOMETHING' }
...
```

Reply:
What do you think?

Reply:
will fix in another pr